spec/controllers/state_file/questions/ny_w2_controller_spec.rb

```diff
@@ -164,6 +164,42 @@
           expect(described_class.show?(intake)).to eq true
         end
       end
+
+      context "StateAbberviationCd is blank or missing" do
+        let(:direct_file_xml) do
+          xml = super()
+          xml.search("W2StateTaxGrp StateAbbreviationCd").each { |node| node.remove }
+          xml
+        end
+
+        it "returns true" do
+          expect(described_class.show?(intake)).to eq true
+        end
+      end
+
+      context "StateIncomeTaxAmt is greater than StateWagesAmt" do
+        let(:direct_file_xml) do
+          xml = super()
+          xml.at("StateIncomeTaxAmt").content = "9000"
+          xml
+        end
+
+        it "returns true" do
+          expect(described_class.show?(intake)).to eq true
+        end
+      end
+
+      context "LocalIncomeTaxAmt is greater than LocalWagesAndTipsAmt" do
+        let(:direct_file_xml) do
+          xml = super()
+          xml.at("LocalIncomeTaxAmt").content = "9000"
+          xml
+        end
+
+        it "returns true" do
+          expect(described_class.show?(intake)).to eq true
+        end
+      end
```

Comment:
Could be good to have a spec where #show returns false also.

Reply:
Good call - adding it now

Reply:
We have some already on L17, L50 & L112,